go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/open-policy-agent/opa
 
-go 1.22.7
-
-toolchain go1.23.1
+go 1.23.1
```

Comment:
This has repercussions for lib users. Too early to bump?

Reply:
Should be fine as our "contract" says we lag one version behind and 1.24 is out now.

Reply:
Yeah, I was thinking the same when I did the change, but turned unsure. We just need to remember to make a note of it in the release notes.